lib/src/center.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:turf/helpers.dart';
+
+List<double> center(BBox bBox) {
```

Comment:
In this library, we are not using List<double> to represent coordinates, but the type `Position`, defined in lib/src/geojson.dart.

If you could refactor this function to return:
```dart
return Position.named(
    lng: (bBox.lng1 + bBox + lng2) / 2,
    lat: (bBox.lat1 + bBox.lat2) / 2,
);
```

also very **_important_**:
GeoJSON coordinates are stored in this order (!): [**lng**, **lat**, alt].

Reply:
I'll do the necessary changes.

Reply:
Changed to return Position as requested.